### PR TITLE
chore: remove react-icons dependency

### PR DESCRIPTION
**Pull request by shishkin**
by inlining the HiChevronRight SVG

**Description**

Changes proposed in this pull request:

- remove react-icons dependency in favor of a single static SVG.

**Related issue(s)**

Addresses comment in https://github.com/asyncapi/asyncapi-react/issues/690#issuecomment-1398179446.

### CODE REVIEW

**Comment by magicmatatjahu** [comment]
@shishkin The icon is much bigger than the previous one. Check this one:

New:

image

Old:

image

I have to check old svg, because probably the new one have difference props and by this we have bigger chevron.

File: library/src/components/CollapseButton.tsx [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+  >
+    <path
+      fill="none"
+      stroke="currentColor"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      strokeWidth="1.5"
+      d="m8.25 4.5l7.5 7.5l-7.5 7.5"
+    />
+  </svg>
```
Comment (magicmatatjahu) [inline library/src/components/CollapseButton.tsx:25]: Change it to the:

```suggestion
  <svg 
    stroke="currentColor" 
    fill="currentColor" 
    strokeWidth="0" 
    viewBox="0 0 20 20" 
    height="1em" 
    width="1em" 
    xmlns="http://www.w3.org/2000/svg"
    {...props}
  >
    <path 
      fillRule="evenodd" 
      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z" 
      clipRule="evenodd"
    />
  </svg>
```

Comment (shishkin) [reply library/src/components/CollapseButton.tsx:25]: Yes, react-icons [used the v1](https://github.com/react-icons/react-icons/blob/master/packages/react-icons/src/icons/index.ts#L417-L446) of HeroIcons and I added v2.

**Review by magicmatatjahu — DISMISSED** [review]

**Comment by magicmatatjahu** [comment]
@shishkin Sorry, Could you run `npm run lint:fix`?

**Review by magicmatatjahu — APPROVED** [review]

**Pull request merged**